apps/web-giddh/src/app/invoice/proforma/proforma-list.component.scss

```diff
@@ -392,7 +392,7 @@ span.icon-advance-filter {
         padding-top: 10px;
         padding-bottom: 0;
     }
-    .smallLoading {
+    .small-loading {
```

Comment:
Can we merge these two styles as they both are for the same selector?

Reply:
no they are same selector but for different media query.